ci-operator/jobs/redhat-operator-ecosystem/playground/redhat-operator-ecosystem-playground-cvp-ocp-4.4-periodics.yaml

```diff
@@ -51,6 +51,9 @@ periodics:
         name: result-aggregator
         readOnly: true
     serviceAccountName: ci-operator
+    env:
+    - name: BASE_DOMAIN
+      value: cpaas-ci.devcluster.openshift.com
```

Comment:
This is not how step parameters work. You need to specify the parameters in ci-operator configs these jobs are generated from, not in jobs. There is an example at the end of this section of the doc: https://docs.ci.openshift.org/docs/architecture/step-registry/#parameters, or there are plenty of examples in `openshift/release` (like https://github.com/openshift/release/blob/18576c185349b959b64c64a789b3c16fb8fa7872/ci-operator/config/openshift/ocs-operator/openshift-ocs-operator-master.yaml#L99-L116)